azext_iot/operations/events3/_events.py

```diff
@@ -127,7 +127,15 @@ def _output_msg_kpi(msg):
         event_source = {'event': {}}
 
         event_source['event']['origin'] = origin
-        event_source['event']['payload'] = str(next(msg.get_data()), 'utf8')
+
+        payload = ''
```

Comment:
`str(next(msg.get_data()))` was crashing the vs code terminal when --data was null.  I'm checking for data before calling next.